README.md

```diff
@@ -246,7 +247,7 @@ This method accepts the same parameters as `user.getConnectData()` except for `r
 ```javascript
 user.connect({
   // additional options
-}).onConnection(account => {
+}).onConnection((account, message) => {
```

Comment:
Do we still have the new `message` param?

Reply:
Discussed in thread, reverted.